### PR TITLE
made the sites copyright year dyanamic

### PR DESCRIPTION
**Pull request by agesa3**
## Updated the copyright year to be dynamic ##
Made the copyright year dynamic rather than static,hence the year will be updating itself automatically.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -225,7 +225,8 @@ <h5 class="grey-text text-lighten-4 an-1 ">
                         <div class="footer-copyright">
                             <div class="container swiper-no-swiping">
                                 <div>
-                                    Copyright 2019 <a class="blue-text text-lighten-3 text-underLine"
+                                   Copyright <span class="an-1">{{ new Date().getFullYear() }}</span>
```
Comment (frankfliu) [inline index.html:228]: Thank for your contribution. 

I think the single year copy right should state the year which the first time it published. I think this should not be dynamic

Comment (agesa3) [reply index.html:228]: what about having something like 2019-current year?

Comment (frankfliu) [reply index.html:228]: That should work.

Comment (agesa3) [reply index.html:228]: Ok let me make that change then.

**Review by frankfliu — APPROVED** [review]

**Pull request merged**